### PR TITLE
CORTX-30701 Community Build steps improvements. 

### PR DESCRIPTION
**Pull request by shailesh-vaidya**
### Describe your changes in brief
Improved Community build steps to make it easier to follow. 

- Removed obsolete operating system and Kernel references. 
- Improved clone command to not clone cortx-rgw separately since it's added as a submodule in cortx repo.
- Improved command to be used for checking component branches. 
- Updated docker images list command.

### Changes
 - [x] Why is this change required? What problem does it solve?  Improved build steps.
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [x] Please describe in detail how you tested your changes.  Tested on dev system.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [x] tested locally
 - [ ] added new dependencies
 - [x] updated the docs
 - [ ] added a test

-----
[View rendered doc/community-build/docker/cortx-all/README.md](https://github.com/shailesh-vaidya/cortx/blob/community-build-patch-1/doc/community-build/docker/cortx-all/README.md)

### CODE REVIEW

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

File: doc/community-build/docker/cortx-all/README.md [hunk 1]
```diff
@@ -39,27 +27,23 @@ This document provides step-by-step instructions to build required binaries and
 
 1. Run the following command to clone the CORTX repository:
     ```
-    cd /mnt && git clone https://github.com/Seagate/cortx --recursive --depth=1 && cd /mnt/cortx && git clone https://github.com/Seagate/cortx-rgw
+    cd /mnt && git clone https://github.com/Seagate/cortx --recursive --depth=1
     ```
     
-2.  Please Checkout **main** branch for generating CORTX packages. Use below command for checkout. 
+2.  Please checkout **main** branch for generating CORTX packages. Use below command for checkout.
```
[the remark this answers is not in the document]
Comment (shailesh-vaidya) [reply doc/community-build/docker/cortx-all/README.md]: Done. Please review

File: doc/community-build/docker/cortx-all/README.md [hunk 2]
```diff
@@ -81,24 +65,27 @@ This document provides step-by-step instructions to build required binaries and
...
+    Ensure that container is started and we can download RELEASE.INFO
     ```
     docker ps 
     curl -L http://$HOSTNAME/RELEASE.INFO
-    git clone https://github.com/Seagate/cortx-re && cd cortx-re/docker/cortx-deploy/
     ```
+    
 
-    - If you run build.sh by $HOSTNAME then here we need change docker-compose.yml and add below extra_hosts in that docker compose for all the services like below.
+7. Clone [cortx-re](https://github.com/Seagate/cortx-re) repository using below commands,
```
[the remark this answers is not in the document]
Comment (shailesh-vaidya) [reply doc/community-build/docker/cortx-all/README.md]: Done. Please review

File: doc/community-build/docker/cortx-all/README.md [hunk 3]
```diff
@@ -115,32 +102,27 @@ This document provides step-by-step instructions to build required binaries and
 
 8. After verifying docker compose then run the build.sh file where your cortx-all folder is located.
 
-    - Use below command to build cortx-all image using rocky linux:
+    - Use below command to build cortx-all image. Enure you are in `cortx-re/docker/cortx-deploy/` directory:
```
[the remark this answers is not in the document]
Comment (shailesh-vaidya) [reply doc/community-build/docker/cortx-all/README.md]: Done. Please review

File: doc/community-build/docker/cortx-all/README.md [hunk 4]
```diff
@@ -81,24 +65,27 @@ This document provides step-by-step instructions to build required binaries and
     ```
     docker run --name release-packages-server -v /var/artifacts/0/:/usr/share/nginx/html:ro -d -p 80:80 nginx
     ```
-
-7. We need to clone cortx repo, use below commands for same.
-
+    Ensure that container is started and we can download RELEASE.INFO
```
[the remark this answers is not in the document]
Comment (shailesh-vaidya) [reply doc/community-build/docker/cortx-all/README.md:68]: Downloading RELEASE.INFO is to just test that nginx container is running it's not required in our workflow. Maybe we can use. 
`Ensure that container is started and validate by downloading RELEASE.INFO`

[the remark this answers is not in the document]
Comment (mukul-seagate11) [reply doc/community-build/docker/cortx-all/README.md:68]: OK

File: doc/community-build/docker/cortx-all/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mukul-seagate11) [reply doc/community-build/docker/cortx-all/README.md]: Enure should be Ensure

**Review by mukul-seagate11 — APPROVED** [review]
looks good, hence merging

**Pull request merged**